barcode/_version.py

```diff
@@ -0,0 +1,2 @@
+from __future__ import unicode_literals
+__version__ = '14.0'
```

Comment:
Minor note: as you can probably tell, I've generally been using the mag-year as the version number in Uber and Uber-related repos.  Now that we're switching over to calendar-year numbering (e.g. this upcoming MAGFest is "MAGFest 2016" and we're not calling it "MAGFest 14") I wonder if we should do the same thing for our version numbering.  Ubuntu does this, e.g. I'm running Ubuntu 12.04 which means it was the version of Ubuntu released in April 2012.

Not that this matters in any meaningful way, it was just an idle though.